packages/devel/glibc/package.mk

```diff
@@ -134,6 +134,17 @@ post_makeinstall_target() {
 
 # remove locales and charmaps
   rm -rf $INSTALL/usr/share/i18n/charmaps
+  if [ -n "$GLIBC_LOCALES" ]; then
+    mkdir -p $INSTALL/usr/lib/locale
+    for locale in $GLIBC_LOCALES; do
+      echo ">>> install inputfile $(echo $locale | cut -f1 -d ".") with charmap $(echo $locale | cut -f2 -d ".") as $locale <<<"
+      I18NPATH=../localedata \
+      $ROOT/$TOOLCHAIN/bin/localedef \
```

Comment:
master does not use $ROOT anymore, also you need to add the eglibc-localedef:host package to get localedef in the toolchain. Most Linux distributions ship localedef by default though, so I guess you could remove the path entirely and assume that the host system has it.